components/recurring-contributions/RecurringContributionsCard.js

```diff
@@ -73,15 +69,13 @@ const RecurringContributionsCard = ({ collective, status, contribution, createNo
   const { formatMessage } = useIntl();
   const statusTag = `${status} contribution`;
   const buttonText = status === 'ACTIVE' ? formatMessage(messages.manage) : formatMessage(messages.activate);
-  const userIsLoggedInUser = props.LoggedInUser.collective.slug === props.router.query.slug;
-  // const userIsAdminOfCollectiveOrOrg
-  const userIsAdmin = userIsLoggedInUser; // || userIsAdminOfCollectiveOrOrg
+  const isAdmin = LoggedInUser && LoggedInUser.canEditCollective(account);
 
   return (
     <StyledCard onMouseEnter={() => setHovering(true)} onMouseLeave={() => setHovering(false)} {...props}>
       <Container style={{ background: getBackground(collective) }} backgroundSize="cover" height={100} px={3} pt={26}>
         <Container border="2px solid white" borderRadius="25%" backgroundColor="white.full" width={68}>
-          <LinkCollective collective={collective}>
+          <LinkCollective collective={collective} type={collective.type}>
```

Comment:
I don't see any `type` prop listed for `LinkCollective`, what's its purpose?

Reply:
There was this error. I think I have misunderstood it then?
![image](https://user-images.githubusercontent.com/37520401/84918662-cd1e2f80-b0b8-11ea-9c63-7b63d47bd6ac.png)

Reply:
The missing prop is `collective.type`, not `type`. May be the field is not fetched in the GraphQL query? In any case I'd be surprised if this change removed the warning

Reply:
You were right! It was adding the graphql field, not the prop, that fixed this.